cmd/ko/README.md

```diff
@@ -0,0 +1,146 @@
+# `ko`
+
+## What is `ko`?
+
+`ko` is an **experimental** CLI to support rapid development of Go containers
+with Kubernetes.
+
+## Installation
+
+`ko` can be installed via:
+
+```shell
+go install github.com/google/go-containerregistry/cmd/ko
+```
+
+## The `ko` Model
+
+`ko` is built around a very simple extension to Go's model for expressing
+dependencies using import paths. In Go, dependencies are expressed via
+blocks like:
+
+```go
+import (
+    "github.com/google/go-containerregistry/authn"
+    "github.com/google/go-containerregistry/name"
+)
+```
+
+Similarly (as you can see above), Go binaries can be references via import
+paths like `github.com/google/go-containerregistry/cmd/ko`.
+
+With `ko`, we simply replace image references within Kubernetes yaml
+files with the import paths to the Go binaries that should be containerized
+and substituted there. For example, you might use the following in a
+Kubernetes `Deployment` resource:
+
+```yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: hello-world
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: hello-world
+        # This is the importpath for the Go binary to build and run.
+        image: github.com/examples/http/cmd/helloworld
+        ports:
+        - containerPort: 8080
+```
+
+**This extension of the Go convention enables us to create a tool with an
+extremely fast development cycle and effectively zero configuration.**
+
+## Usage
+
+`ko` can act as a drop-in replacement for the `go` CLI, to which it passes
+through all of the top-level `go foo` commands.
+
+`ko` also exports three commands that interact with Kubernetes resources.
+
+### `ko delete`
+
+`ko delete` simply passes through to `kubectl delete`, as with the `go`
```

Comment:
Likewise for `ko delete` -- what benefit do I get from running `ko delete foo.yaml` over `kubectl delete foo.yaml`?

Reply:
I find the symmetry with `ko apply` useful, I have a stronger bias to keep this.